addons/website/static/src/js/menu/content.js

```diff
@@ -327,6 +330,46 @@ var PagePropertiesDialog = weWidgets.Dialog.extend({
         }
     },
 
+
+    /**
+     * Allows the given input to propose existing groups.
+     *
+     * @param {ServicesMixin|Widget} self - an element capable to trigger an RPC
+     * @param {jQuery} $input
+     */
+
+    autocompleteWithGroups: function ($input) {
+        var self = this;
+        $input.autocomplete({
+            source: (request, response) => {
+                return this._rpc({
+                    model: 'res.groups',
+                    method: 'search_read',
+                    args: [[['name', 'ilike', request.term]], ['display_name']],
+                    kwargs: {
+                        limit: 15,
+                    },
+                }).then(function (founds) {
+                    var finds = _.map(founds, function (f) {
+                        return { 'id': f.id, 'label':f.display_name };
+                    });
+                    response(finds);
+                });
```

Comment:
- Only use `function` for class methods.
- You can use standard `map` function
- Your linter should tell you about wrong spaces here :p 

```javascript
.then(groups => response(groups.map(g => {'id': g['id'], 'label': g['display_name']});
```
(or multiline if you want)